Add pages on visa information and opportunity grants (#17)
djangocon/2023.djangocon.eu: _pages/visas.md

---
description: Information about obtaining a Visa to visit DjangoCon Europe
heading: Visa Information
layout: default
permalink: /visas/
published: true
title: Visa Information
---

## Do I need a visa to attend DjangoCon Europe?

If you’re planning to attend DjangoCon Europe, first check whether you need a visa to 

> visit the UK. Fill in the form on [this page](https://www.gov.uk/check-uk-visa/y) to check 
> whether you need a visa.  A standard UK visitor visa costs £100, and may take around 6 weeks to process.  You can find more information on visa requirements for visiting the UK 
> here.

Reviewer: As a general rule we should always use a link label that’s descriptive of what the link takes the user to, so screen reader users can more easily navigate the page link-by-link.

We should probably link to the parent page (`https://www.gov.uk/check-uk-visa`), so people understand where in the site they are and can see related resources.

```suggestion
visit the UK. Fill in the official [Check if you need a UK visa](https://www.gov.uk/check-uk-visa) form to check. 
A standard UK visitor visa costs £100, and may take around 6 weeks to process. You can find more information on visa requirements for visiting the UK here.
```

Is the last "here" missing a link?
